src/views/open-api-docs-view/index.tsx

```diff
@@ -62,6 +64,14 @@ function OpenApiDocsView({
 						badgeText={releaseStage}
 						serviceProductSlug={serviceProductSlug}
 						statusIndicatorConfig={statusIndicatorConfig}
+						versionSwitcherSlot={
+							versionSwitcherProps ? (
```

Comment:
At the view level, it's a relatively simple task of render a `VersionSwitcher` with `versionSwitcherProps` into the overview component (if those props are present, which they'll only be for OpenAPI docs with 2 or more separate versions).